### PR TITLE
Update website preview

### PR DESCRIPTION
**Pull request by ktranada**
* Updated modal portal to accept node prop
* Updated panel navigator
* Updated previews to share base configuration
* Bump react types

### CODE REVIEW

File: src/components/modal/ModalPortal.tsx [hunk 1]
```diff
@@ -209,9 +218,11 @@ export class ModalPortal extends React.Component<Props, State> {
     const { isClosing } = this.state;
     const {
       children,
+      clickThroughWrapper,
       closeOnBackgroundClick,
       closeOnEscape,
       onClose,
+      node,
```
Comment (mcorrigan89) [inline src/components/modal/ModalPortal.tsx]: What is the node for here?

File: src/components/panel/nav/PanelNavigator.tsx [hunk 2]
```diff
@@ -67,34 +73,62 @@ const navItemRules = css({
...
-// @TODO: NOT USED CURRENTLY
-// const renderNavItems = navItems => {
-//   if (navItems instanceof Array) {
-//     return navItems.map(({ content, key, handleOnClick }) => (
-//       <PanelNavItem key={key} content={content} handleOnClick={handleOnClick} />
-//     ));
-//   }
-//   return navItems;
-// };
+const renderNavItems = navItems => {
```
Comment (mcorrigan89) [inline src/components/panel/nav/PanelNavigator.tsx]: Do we have a typed interface for what a nav item should be?

File: src/components/modal/ModalPortal.tsx (hunk 1 above)
Comment (ktranada) [reply src/components/modal/ModalPortal.tsx]: node is for if went want to append  the portal to something other than body. IE in the angular + react intersection, we don't want to put the modal to the body b/c page changes won't remove it

File: src/components/panel/nav/PanelNavigator.tsx (hunk 2 above)
Comment (ktranada) [reply src/components/panel/nav/PanelNavigator.tsx]: I'll add that

**Pull request merged**